### PR TITLE
argparse: init at 2.6

### PR DESCRIPTION
**Pull request by 2gn**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/argparse/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "argparse";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    owner = "p-ranav";
+    repo = "${pname}";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/argparse/default.nix:9]:
```suggestion
    repo = "argparse";
```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -17043,6 +17043,8 @@ with pkgs;
   aravis = callPackage ../development/libraries/aravis { };
 
   arb = callPackage ../development/libraries/arb {};
+
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:17046]:
```suggestion

```

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -17043,6 +17043,8 @@ with pkgs;
   aravis = callPackage ../development/libraries/aravis { };
 
   arb = callPackage ../development/libraries/arb {};
+    
+  argparse = callPackage ../development/libraries/argparse {};
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:17047]:
```suggestion
  argparse = callPackage ../development/libraries/argparse { };
```

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (2gn) [reply pkgs/top-level/all-packages.nix:17046]: I would like to accept these changes, but GitHub says "This diff has recently been updated. Refresh and try again." and I can't merge it. Is it okay if I force-push the changes you suggested in your review?
image

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:17046]: thats a but in github. Yeah, sure. We force push all the time.

**Review by ocfox — APPROVED** [review]
LGTM

**Pull request merged**